test/precision_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule PrecisionTest do
+  use ExUnit.Case,
+    async: false
+
+  import Ecto.Query
+
+  alias ExAudit.Test.{Repo, User, Version, Util}
+
+  setup_all do
+    current = Application.get_env(:ex_audit, :precision)
+
+    Application.put_env(:ex_audit, :precision, :microsecond)
+
+    on_exit(fn -> Application.put_env(:ex_audit, :precision, current) end)
+  end
+
+  test "adjust the precision to `second` in the field `recorded_at`" do
```

Comment:
I don't see how this tests that, you've set the precision to `:microsecond` in the `setup_all` block.